app/scenes/Settings/Drawio.tsx

```diff
@@ -0,0 +1,98 @@
+import { find } from "lodash";
+import { observer } from "mobx-react";
+import { BuildingBlocksIcon } from "outline-icons";
+import * as React from "react";
+import { useForm } from "react-hook-form";
+import { useTranslation, Trans } from "react-i18next";
+import { IntegrationType } from "@shared/types";
+import Integration from "~/models/Integration";
+import Button from "~/components/Button";
+import Heading from "~/components/Heading";
+import { ReactHookWrappedInput as Input } from "~/components/Input";
+import Scene from "~/components/Scene";
+import Text from "~/components/Text";
+import useStores from "~/hooks/useStores";
+
+type FormData = {
+  url: string;
+};
+
+function Drawio() {
+  const { integrations } = useStores();
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    integrations.fetchPage({
+      limit: 100,
```

Comment:
```suggestion
      limit: 100,
      type: IntegrationType.Embed,
```

Reply:
:point_down: seems better?

```suggestion
    service: "diagrams",
```

Reply:
Fixed!